scripts/basic-workflow-tests.sh

```diff
@@ -162,6 +163,16 @@ test_convert() {
     ilab model convert
 }
 
+test_evaluate() {
+    task Evaluating a model
+
+    if [ "$EVAL" -eq 0 ]; then
+      export INSTRUCTLAB_EVAL_MMLU_MIN_TASKS=true
+      export HF_DATASETS_TRUST_REMOTE_CODE=true
+      ilab model evaluate --model models/instructlab/granite-7b-lab --benchmark mmlu
```

Comment:
Could you change it to test the output of `ilab model train` ? It seems like that would be more representative of an end-to-end user flow, right?

Reply:
Do you want that in this function or in `test_exec()` where the full workflow happens?

Reply:
in here